swift/apple/FirezoneKit/Sources/FirezoneKit/Features/AskPermissionView.swift

```diff
@@ -34,13 +34,13 @@ public final class AskPermissionViewModel: ObservableObject {
     self.sessionNotificationHelper = sessionNotificationHelper
 
     tunnelStore.$tunnelAuthStatus
-      .filter { $0.isInitialized }
+      .filter { $0 != nil }
       .sink { [weak self] tunnelAuthStatus in
         guard let self = self else { return }
 
         Task {
           await MainActor.run {
-            if case .noTunnelFound = tunnelAuthStatus {
+            if case .noManagerFound = tunnelAuthStatus {
```

Comment:
I renamed the instance of `NETunnelProviderManager` from `tunnel` to `manager` because tunnel is a bit of an overloaded term and a manager is what it is.